packages/kit/src/core/adapt/prerender.js

```diff
@@ -86,8 +87,11 @@ export async function prerender({ cwd, out, log, config, build_data, force }) {
 		}
 	});
 
-	/** @param {string} path */
-	async function visit(path) {
+	/**
+	 * @param {string} path
+	 * @param {string} parent
+	 */
+	async function visit(path, parent) {
```

Comment:
`parent_path` might be a clearer name? Or could update the JSDoc to clarify that it's a path

Reply:
I feel like it's obvious from usage. If you go by the name alone then you might think that the `parent_path` of `/foo/bar` was `/foo`, whereas if you scan down to where it's used, it becomes clear what it does